### PR TITLE
Izen's week-1 JS exercises 

### PR DESCRIPTION
**Pull request by Eisen9**
Week-1 JS exercises.

### CODE REVIEW

File: week-1-practise/level-1/1-syntax-error-1.js [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
 var result = addNumbers(1, 3, 4);
 console.log(result); // Expected result 8 - Run the program and make sure it works
 
-
 // Answer these questions:
-// 1. How many parameters does the function take?
-// 2. What is the function name?
-// 3. Where is the function called? with what parameters?
\ No newline at end of file
+// 1. How many parameters does the function take? 3
```
Comment (PaulinaGrabania) [inline week-1-practise/level-1/1-syntax-error-1.js]: great answer!  hehe

File: week-1-practise/level-1/7-logic-error-4.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ function multiply(a, b, c) {
   return;
 }
 
-var result = multiply(1, 3, 4);
+var result = (multiply = 1 * 3 * 4);
```
Comment (Ashaghel) [inline week-1-practise/level-1/7-logic-error-4.js]: interesting way to think

File: week-1-practise/level-1/1-syntax-error-1.js [hunk 3]
```diff
@@ -1,14 +1,15 @@
 // There is a syntax error in this code. Fix it.
 
-function addNumbers(a b c) {
-    return a + b + c;
+function addNumbers(a, b, c) {
```
Comment (folukeOpenuni) [inline week-1-practise/level-1/1-syntax-error-1.js]: well done, you spotted the mistake

File: week-1-practise/level-1/1-syntax-error-1.js [hunk 4]
```diff
@@ -1,14 +1,15 @@
...
 
-
 // Answer these questions:
-// 1. How many parameters does the function take?
-// 2. What is the function name?
-// 3. Where is the function called? with what parameters?
\ No newline at end of file
+// 1. How many parameters does the function take? 3
+// 2. What is the function name? addNumbers
+// 3. Where is the function called? with what parameters? the function is called on line 7, it is assigned to the variable result. It  is called with 0, 1, 2 parameters
```
Comment (Ibraham00) [inline week-1-practise/level-1/1-syntax-error-1.js]: Good

File: week-1-practise/level-1/2-syntax-error-2.js [hunk 5]
```diff
@@ -1,8 +1,9 @@
 // There is a syntax error in this code. Fix it.
 
-function addNumbers(a, b, c)
-return a + b + c;
-
-
+function addNumbers(a, b, c) {
+  return a + b + c;
```
Comment (NawalC) [inline week-1-practise/level-1/2-syntax-error-2.js]: Great Job, Izen!

File: week-1-practise/level-1/2-syntax-error-2.js [hunk 6]
```diff
@@ -1,8 +1,9 @@
 // There is a syntax error in this code. Fix it.
 
-function addNumbers(a, b, c)
-return a + b + c;
-
-
+function addNumbers(a, b, c) {
```
Comment (MariyaPo) [inline week-1-practise/level-1/2-syntax-error-2.js]: Good parameters

File: week-1-practise/level-1/10-function-output-3.js [hunk 7]
```diff
@@ -1,6 +1,11 @@
...
-    // Write the body of this function to concatenate three words together
+  return firstWord.concat(secondWord + thirdWord);
+
+  // Write the body of this function to concatenate three words together
 }
 
-var result = concatenate('code', 'your', 'future');
-console.log(result); // expected result = "codeyourfuture"
\ No newline at end of file
+var result = concatenate("code", "your", "future");
```
Comment (folukeOpenuni) [inline week-1-practise/level-1/10-function-output-3.js]: I thought single quote and double quote does the same thng

File: week-1-practise/level-1/2-syntax-error-2.js [hunk 8]
```diff
@@ -1,8 +1,9 @@
...
+  return a + b + c;
+}
 var result = addNumbers(1, 3, 4);
 console.log(result); // Expected result 8 - Run the program and make sure it works
```
Comment (MariyaPo) [inline week-1-practise/level-1/2-syntax-error-2.js]: result should be here

File: week-1-practise/level-1/7-logic-error-4.js (hunk 2 above)
Comment (PaulinaGrabania) [reply week-1-practise/level-1/7-logic-error-4.js]: I think the same

File: week-1-practise/level-1/1-syntax-error-1.js [hunk 9]
```diff
@@ -1,14 +1,14 @@
 // There is a syntax error in this code. Fix it.
 
-function addNumbers(a b c) {
-    return a + b + c;
+function addNumbers(a, b, c) {
```
Comment (maaly11) [inline week-1-practise/level-1/1-syntax-error-1.js]: I like that :+1:

File: week-1-practise/level-1/1-syntax-error-1.js (hunk 4 above)
Comment (folukeOpenuni) [reply week-1-practise/level-1/1-syntax-error-1.js]: nicelly done

File: week-1-practise/level-1/1-syntax-error-1.js (hunk 1 above)
Comment (PaulinaGrabania) [reply week-1-practise/level-1/1-syntax-error-1.js]: Who else think that he done a great job?

File: week-1-practise/level-1/2-syntax-error-2.js (hunk 5 above)
Comment (efahmed) [reply week-1-practise/level-1/2-syntax-error-2.js]: good job , I like it

Comment (folukeOpenuni) [reply week-1-practise/level-1/2-syntax-error-2.js]: cool

File: week-1-practise/level-1/8-function-output-1.js [hunk 10]
```diff
@@ -1,12 +1,12 @@
 function getNumber() {
-    return Math.random() * 10;
+  return Math.random() * 10;
```
Comment (Abdoulrazack95) [inline week-1-practise/level-1/8-function-output-1.js]: Good men

File: week-1-practise/level-1/1-syntax-error-1.js (hunk 1 above)
Comment (boubacardllo) [inline week-1-practise/level-1/1-syntax-error-1.js]: Good job i like the way that you break it down

**Pull request closed without merging**